db/seeds.rb

```diff
@@ -1,7 +1,89 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
```

Comment:
it will be good to clean db before seed

Recipe.destroy_all
Ingidient.destroy_all
...